repos/system_upgrade/el8toel9/actors/checkcustomnetworkscripts/libraries/customnetworkscripts.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+from leapp import reporting
+
+CUSTOM_NETWORK_SCRIPTS = [
+    "/sbin/ifup-local",
+    "/sbin/ifup-pre-local",
+    "/sbin/ifdown-local",
+    "/sbin/ifdown-pre-local",
+]
+DOC_URL = "https://red.ht/upgrading-RHEL-8-to-RHEL-9-network-scripts"
+
+
+def generate_report():
+    """ Generate reports informing user about possible manual intervention requirements """
+
+    # Show documentation url if custom network-scripts detected
+    title = "custom network-scripts detected"
+    summary = (
+        "RHEL 9 does not support the legacy network-scripts package that was"
+        " deprecated in RHEL 8, and the custom network-scripts have been"
+        " detected. In order to proceed with the upgrade, you can move the"
+        " custom network-scripts, for example, to the `/opt/network-scripts/`"
```

Comment:
The advise to move the scripts to `/opt/network-scripts/` contradicts the advise to move to dispatcher scripts (which should be the only recommended way). The summary does not seem to be the right place to mentioned the remediation step, so I would remove this sentence here.

Reply:
I removed the "advice".